ext/linux/osq_lock.h

```diff
@@ -338,12 +328,13 @@ void osq_unlock(uint64_t *osq, unsigned long cpu_number)
 
 unsigned long __attribute__((noinline)) lock_acquire (uint64_t *lock, unsigned long threadnum)
 {
-    /* TODO/BUG: need to implement mutex slow path like
+    /* TODO: need to implement mutex slow path like
      * __mutex_lock_common() and mutex_optimistic_spin()
      * in linux/kernel/locking/mutex.c
      */
     while (!osq_lock(lock, threadnum)) {
-        usleep(random_sleep);
+        /* maximum usleep time: 5us */
```

Comment:
Please leave a more detailed comment to explain how this is different from the Linux kernel.